pyturbo/systems/nozzle/simple_nozzle.py

```diff
@@ -0,0 +1,58 @@
+from cosapp.base import System
+from pyturbo.ports import FluidPort
+
+
+class SimpleNozzle(System):
+    def setup(self):
+
+        # Define the ports
+        self.add_input(FluidPort, "F_in")
+        self.add_output(FluidPort, "F_out")
+
+        # Define inwards
+        self.add_inward("S_inlet", value=3.14159)
+        self.add_inward("S_outlet", value=2.0)
+
```

Comment:
respect conventional name when ever possible

Reply:
Component has been removed as it was useless